collector.js

```diff
@@ -101,11 +105,11 @@ class AlAzureCollector {
         var stats = this._collectionStats;
         
         if (messages && messages.length > 0) {
-            alcollector.AlLog.buildPayload(this._hostId, this._sourceId, hm, messages, formatFun, function(err, payload){
+            alcollector.AlLog.buildPayload({hostId: this._hostId, sourceId: this._sourceId, hostmetaElems: hm, content: messages, parseCallback: formatFun, filterJson: this._filterJson, filterRegexp: this._filterRegex}, function(err, data){
```

Comment:
could you please refactor params into a separate var and fix this very long code string.

Reply:
Done